.env

```diff
@@ -0,0 +1,7 @@
+MYSQL_ROOT_PASSWORD=root
```

Comment:
```suggestion
MYSQL_ROOT_PASSWORD=
```

Reply:
not fixed, push .env.template here with actual values removed
use git rm --cached .env to remove .env from remote while keeping it locally, and ensure it's not committed again by adding it to .gitignore